modules/Bio/EnsEMBL/Variation/DBSQL/PublicationAdaptor.pm

```diff
@@ -226,7 +226,170 @@ sub fetch_all_by_Variation {
 
 }
 
+=head2 fetch_sources_by_pmid
```

Comment:
You cannot use fetch for your method name. Using fetch implies you get back Publication objects. We need to choose a name which better explains the return value. get_variants_to_sources_by_pmid?